src/openzaak/components/documenten/tests/test_cmis_queries.py

```diff
@@ -54,6 +54,9 @@ def test_chained_filter(self):
         self.assertEqual(list(eios), [])
 
     def test_all(self):
```

Comment:
Not sure why this is failing now :thinking: https://github.com/open-zaak/open-zaak/runs/8184171702?check_suite_focus=true#step:11:310